modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -499,18 +532,16 @@ namespace {{packageName}}.Client
         /// </summary>
         /// <param name="filename">Filename</param>
         /// <returns>Filename</returns>
-        public static string SanitizeFilename(string filename)
-        {
+        public static string SanitizeFilename(string filename) {
+            log.Trace("start");
             Match match = Regex.Match(filename, @".*[/\\](.*)$");
 
-            if (match.Success)
-            {
+            if (match.Success) {
                 return match.Groups[1].Value;
-            }
-            else
-            {
+            } else {
                 return filename;
             }
+            log.Trace("end");
```

Comment:
This is dead code. The trace needs to be duplicated in each block above, or the if/else would need to be rewritten to store in a variable and return after this statement.